tests/test_create_observation/test_create_quantity_observation.py

```diff
@@ -25,11 +27,14 @@ async def test_create_quantity_observation_success(mocker):
 
 
 @pytest.mark.asyncio
+@pytest.mark.dependency(name="quantity-failure", depends=["quantity-success"])
 @pytest.mark.quantity
 @pytest.mark.sparql
 async def test_create_quantity_observation_failure(mocker):
     """Test"""
 
+    time.sleep(1)
```

Comment:
👀

Reply:
why is this required in a mocked test setup?

Reply:
Needed this one to be the last created by at least a second, because the `observation_date` value doesn't have millisecond resolution, and sorting on that generates the `most_recent` value

Reply:
...but yeah there truly must be a better way